<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PuPil_APP</class>
 <widget class="QMainWindow" name="PuPil_APP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PuPil_APP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Ques_Index_Label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>题目：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Ques_Out_Browser">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Ques_Start_Button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ques_Max_Nums_Label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>输入1-10000：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ques_NextOne_Button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>下一题</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ques_Total_Text">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Ques_YAns_LLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>你的答案：</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ques_Ans_RorN_Label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Ques_Ans_Submit_Button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ques_YAans_Text">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Ques_Scores_Label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>得分：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ques_Cal_Max_Num_Text">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Ques_Set_Cal_Max_Num_Label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>261</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>输入参与计算的最大值(大于等于10):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ques_Cal_Max_Num_Button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>350</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
